scripts/examples/generate_book_cover.py

```
@@ -0,0 +1,144 @@
+            image_data = get_image(image_info['filename'], image_info['subfolder'], image_info['type'])
+            local_filename = f"generated_{book_title.lower().replace(' ', '_')}_{image_info['filename']}"
+            with open(local_filename, "wb") as f:
+                f.write(image_data)
+            print(f"Saved: {local_filename}")
+
+    ws.close()
+
+# --- RUNNING THE CODE ---
+if __name__ == "__main__":
+    # Load your JSON from the file
+    with open("scripts/example_workflow.json", "r") as f:
```

Ruff (0.15.1)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)